### PR TITLE
[IMP] documentation typos in example code #695

### PR DESCRIPTION
**Pull request by RaulRC**
Typo in shape_example

### CODE REVIEW

**Comment by RaulRC** [comment]
Some minimal typos.

**Comment by Corvince** [comment]
Thank you for the heads-up, but just for clarification -- is this really supposed to fix just a single typo? The title and issues sounds like there are several typos (which I am sure is true)

**Comment by RaulRC** [comment]
Yeah you're right, just one found at this very moment. I'm currently using mesa for a project, and I will notice If I find someone else. 

Thanks!

**Comment by Corvince** [comment]
Okay, in that case I will leave this PR open a bit more to collect more typos. 

Please also feel free to ask questions on the [mailing list](https://groups.google.com/forum/#!forum/projectmesa) if you need any help with your project

**Comment by RaulRC** [comment]
Great, Corvince! 

Thanks!

File: examples/shape_example/Readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 A very basic example model to showcase the visulaization on web browser.
```
Comment (rht) [inline examples/shape_example/Readme.md]: Found a typo: s/visulaization/visualization/.

File: examples/shape_example/Readme.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
-A simple grid is dispalyed on browesr with two agents. The example does not
```
Comment (rht) [inline examples/shape_example/Readme.md]: s/dispalyed/displayed/

**Comment by rht** [comment]
I ran  `for f in **/*.md; do aspell check $f; done` on the `examples/` folder. This should be *all of* the remaining typos (their location can be found via `git grep generated`):
- genereated
- cellulat
- unreast
- Citiens
- proceses
- mimicing
- movenment
- pennits

**Comment by RaulRC** [comment]
Thank's a lot! I'll include all of them and pullr again ;D

**Comment by dmasad** [comment]
@RaulRC -- have you had a chance to update the PR with the other typo fixes? If so, make sure to push the update, and this should be good to go.

**Comment by RaulRC** [comment]
> @RaulRC -- have you had a chance to update the PR with the other typo fixes? If so, make sure to push the update, and this should be good to go.

Sorry for the delay guys! In a couple of days I'll update the PR with the list. Too busy this weeks! Apologies!!

**Comment by RaulRC** [comment]
Hi all!

Fixed new typos as displayed in the list above. Ready to go!

File: examples/sugarscape_cg/Readme.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ A simple ecological model, consisting of two agent types: ants, and sugar
 patches.
 
 The ants wander around according to Epstein's rule M:
-- Look out as far as vision pennits in the four principal lattice directions and identify the unoccupied site(s) having the most sugar. The order in which each agent search es the four directions is random.
+- Look out as far as vision pennies in the four principal lattice directions and identify the unoccupied site(s) having the most sugar. The order in which each agent search es the four directions is random.
```
Comment (rht) [inline examples/sugarscape_cg/Readme.md]: `s/pennies/permits/` ?

**Pull request merged**